### ISSUE TITLE
Another book specific issue 

### ISSUE THREAD

**Issue by flintstones-fred** [digitalec/sonus#7]
While processing a book, I am getting an error -
```
Traceback (most recent call last):
  File "/home/user/anaconda3/bin/sonus", line 8, in <module>
    sys.exit(main.main())
  File "/home/user/anaconda3/lib/python3.9/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/user/anaconda3/lib/python3.9/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/user/anaconda3/lib/python3.9/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/user/anaconda3/lib/python3.9/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/user/anaconda3/lib/python3.9/site-packages/sonus/main.py", line 119, in main
    chapterizer.main(tmpdir.name, output_path, generic, offset, ffmpeg_debug)
  File "/home/user/anaconda3/lib/python3.9/site-packages/sonus/chapterizer.py", line 225, in main
    merge_chapter_parts(tmpfiles, output_dir, generic_chapters)
  File "/home/user/anaconda3/lib/python3.9/site-packages/sonus/chapterizer.py", line 152, in merge_chapter_parts
    title = tag_data.get('TALB').text[0]
AttributeError: 'NoneType' object has no attribute 'text'

```
This is the first LOTR book for reference. I am using sonus v0.6.3

**Comment by digitalec** [comment]
It looks like there are several versions of this book. Can you specify which one this was (if you recall)? Narrators I see are Inglis, Serkis, Ensemble Cast and BBC.

**Comment by flintstones-fred** [comment]
So, I could never recall which specific version this was. However, I experiencing the same issue (same exact error, which appears after it has extracted a few parts) with Agatha Christie's "The Regatta Mystery and Other Stories". Now using v0.6.4

**Comment by cerinawithasea** [comment]
I kind of figured out how to get around the book specific error. Im using a terminal called warp, if you get an error run that book again and click warpify when it gets to the error it will give you options instead of stopping. I have included a screenshot where that happened. I chose all and it completed.
This was dumb luck and me messing around with things i have no knowledge of
Bur hey it works
* pause i have to move to my mac ( yes i know i am a mess)
